ambry-store/src/main/java/com.github.ambry.store/PersistentIndex.java

```diff
@@ -76,6 +76,13 @@ public int compare(File o1, File o2) {
       return o1Offset.compareTo(IndexSegment.getIndexSegmentStartOffset(o2.getName()));
     }
   };
+  static final Comparator<IndexEntry> INDEX_ENTRIES_COMPARATOR = new Comparator<IndexEntry>() {
```

Comment:
Renamed to `INDEX_ENTRIES_OFFSET_COMPARATOR`. `BlobStoreCompactor` will use this because it needs ordering by offsets and not keys. However since the index "owns" the index entries, I figured this would be a better place to put the comparator itself.